prov/gni/src/gnix_rma.c

```diff
@@ -250,7 +250,7 @@ int __smsg_rma_data(void *data, void *msg)
 	struct gnix_fid_ep *ep = vc->ep;
 	gni_return_t status;
 
-	if (hdr->flags & FI_REMOTE_CQ_DATA && ep->recv_cq) {
+	if (_GNIX_ALLOW_FI_REMOTE_CQ_DATA(hdr->flags) && ep->recv_cq) {
```

Comment:
The leading underscore is only really needed for function names. Macros don't need them. I know we're being picky but I don't think any of our macros have leading underscores. Let's remove it.